lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -1030,6 +1034,10 @@ public void setNames(String names) {
 		this.names = names;
 	}
 
+	public void setImportarchive(Boolean importarchive) {
+		this.importarchive = importarchive;
+	}
```

Comment:
Use @Setter here, too. You could lombok this whole class really easily, and you may as well :)

Reply:
Yup.  I took a look at Lombokifying the rest of this class and ran screaming.  The last thing I want to do is introduce a Lessons regression into 21.x